lib/govuk_message_queue_consumer/test_helpers/test_consumer.rb

```diff
@@ -3,5 +3,13 @@ class TestConsumer < Consumer
     def publish_message(payload, options)
       exchange.publish(payload, options)
     end
+
+    # call after integration tests finish
+    def purge_and_unbind_queue(routing_key)
+      # purge all messages from the queue and
+      # unbind the queue with the given routing_key
+      queue.purge
+        .unbind(exchange, {routing_key: routing_key})
```

Comment:
I would prefer it, and I did try using `queue.delete` but the operation fails on the dev vm…

I can keep at it and try to find what the problem is